crates/subspace-networking/src/protocols/request_response/handlers/cached_piece_by_index.rs

```diff
@@ -33,6 +36,12 @@ impl GenericRequest for CachedPieceByIndexRequest {
     type Response = CachedPieceByIndexResponse;
 }
 
+impl CachedPieceByIndexRequest {
+    /// Max number of cached pieces to accept per request, equals to the number of source shards in
```

Comment:
Nit: every time I read this, I think “but pieces are 1MB, how can 128 of them fit in a packet?”

This happens in a few comments, but I don’t think it’s significant enough to block merging, because it’s obvious from context.

```suggestion
    /// Max number of cached piece indexes to accept per request, equals to the number of source shards in
```